tests/attr/layer/test_layer_conductance.py

```diff
@@ -161,8 +167,14 @@ def _conductance_test_assert(
             )
 
     def _conductance_reference_test_assert(
-        self, model, target_layer, test_input, test_baseline=None
-    ):
+        self,
+        model: Module,
+        target_layer: Module,
+        test_input: Union[Tensor, Tuple[Tensor, ...]],
+        test_baseline: Optional[
+            Union[int, float, Tensor, Tuple[Union[int, float, Tensor], ...]]
+        ] = None
+    ) -> None:
         layer_output = None
 
         def forward_hook(module, inp, out):
```

Comment:
Since layer conductance has many return types, you will likely need to use "cast" (imported from typing) to specify that the return types will be known. For example, in this function, you would change line 190 to:         
```
attributions, delta = cast(Tuple[Tensor, Tensor], cond.attribute(
            test_input,
            baselines=test_baseline,
            target=target_index,
            n_steps=300,
            method="gausslegendre",
            return_convergence_delta=True,
        ))

```